### ISSUE TITLE
Ponewheel not listed on Play store?

### ISSUE THREAD

**Issue by muellergit** [ponewheel/android-ponewheel#113]
I no longer see Ponewheel listed on the Play store. Is this a temporary glitch?

**Comment by kwatkins** [comment]
Oh wow. Just got this e-mail, no idea why. Something I'll have to @#$ deal with now. 
```This is a notification that your Google Play Publisher account has been terminated.

REASON FOR TERMINATION: Violations of the Developer Program Policies and Developer Distribution Agreement.

Google Play Publisher terminations are associated with developers, and may span multiple account registrations and related Google services.

You can visit the Developer Policy Center to better understand how we enforce Developer Program Policies. If you’ve reviewed the policy and feel this termination may have been in error, please reach out to our policy support team.

Please do not attempt to register a new developer account. We will not be restoring your account at this time.

The Google Play Team```

**Comment by biell** [comment]
Google has been "cracking down" and hopefully you just got caught up by accident.  Onewave got kicked out of the store a while back because of a complaint and I don't think he ever got to the root of it.

I hope this isn't FM pushing back against 3rd party apps.

**Comment by kwatkins** [comment]
Nah it's one of my other apps, false positive malware so they switched off
all my apps. Appealing it now.

**Comment by m-sterspace** [comment]
Is the APK available anywhere for sideloading?

**Comment by beeradmoore** [comment]
Still unresolved?

**Comment by kwatkins** [comment]
Yes. Unless someone wants to host on Google Play, unfortunately they are
taking forever to respond. I'll compile the latest APK and add it to
releases. If anyone has other hosting recommendations it'll be a big help.
Amazon App store is OK other than security risks that come from
sideloading.

The latest APK build is up on  
https://github.com/ponewheel/android-ponewheel/releases/download/v4.4-standalone/app-debug.apk

**Comment by AlexBeggs** [comment]
any word on why Google removed it?

**Comment by kwatkins** [comment]
It wasn't because of the app, one of my other apps got flagged and my
developer account got pulled. It was a stupid mistake, researching a
publically available script and it was in the assets directory of the app.
The way Google works, any and all apps and accounts associated gets the
blacklist. The whole thing is frustrating to say the least.

-- Kevin

**Comment by ghost** [comment]
> Google has been "cracking down" and hopefully you just got caught up by accident. Onewave got kicked out of the store a while back because of a complaint and I don't think he ever got to the root of it.
> 
> I hope this isn't FM pushing back against 3rd party apps.

Hey sorry Ive been gone so long and reviving an old thread, Ive been struggling through health problems. Onewave died because Future Motion DMCA'd the app itself, my developer account is fine. The reasons they stated were for safety. They only did it after I put up a build I believe on the beta branch that I thought had pint support, which ended up only having pint support for my board, whoops. It seems any app that connects to old boards is fine but anything that uses the new handshake will get taken down. I never appealed because then I can be taken to court and I can't have that happen. Its such a shame to me that companies can use laws meant to enforce copyright to take down whatever they want for any reason and as long as the other party is small enough they just get away with it. The app just read values from the board there weren't really any safety concerns with it.

(She btw 💗)

**Comment by biell** [comment]
@Nanoux , I hope your health has improved, and I wish you the best; also I apologize for the incorrect pronoun.

One thing you may want to consider is to share your story with EconLIberties, they are a lobbying organization in support of developers like you.  They support right-to-repair, are anti-monopoly, and I think that casting their very discerning eye at DMCA abuse could only be positive for this community: https://accesstomarkets.org/share-your-story/

**Comment by crocodilus** [comment]
@Nanoux Apologies if this question had already been addressed, but have you considered publishing to F-Droid.org ?

Best wishes!

**Comment by ghost** [comment]
I don't think my app is eligible for F-Droid, I looked into it forever ago and I don't remember what the reason is now, I think it was a dependency on Google Play Services. 

@biell Its no problem! My health is finally doing much better now thank you. The honest answer is that this ordeal has totally made me loose all motivation for my project, I'm using the official app now on my phone again. I still love my board but I have no love of Future Motion and I don't really want to interact with them ever again, I would much rather focus on new 100% independent projects in the future.
